### PR TITLE
Feature/user management

### PR DESCRIPTION
**Pull request by simranjits-cuelogic**
This is request for review and merge code for user management feature including following tasks.

    User can view his/her profile.
    User can update his/her profile details.
    User can change his/her profile photo.
    User can update his/her password.

Thanks

### CODE REVIEW

File: profiles/forms.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    location = forms.ChoiceField(
+        choices = LOCATIONS,
+        label = "Country",
+        initial = 'Please select',
+        widget = forms.Select(),
+        required = True
+        )
+
+    # def clean_avatar(self):
```
Comment (ananthu-nair707) [inline profiles/forms.py]: Code needed? If not remove this.
If yes comment feature and future requirement specifics above piece of code.

File: profiles/views.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required
+from django.db import transaction
+from django.contrib import messages
+from .forms import UserForm, ProfileForm
+
+@transaction.atomic
+def update_profile(request):
+    if request.method == 'POST':
```
Comment (ananthu-nair707) [inline profiles/views.py:21]: Add doc strings for all functions and classes, explaining purpose.

File: profiles/views.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        user_form = UserForm(request.POST, instance = request.user)
+        profile_form = ProfileForm(request.POST, request.FILES, instance = request.user.profile)
+        if user_form.is_valid() and profile_form.is_valid():
+            user_form.save()
+            profile_form.save()
+            messages.success(request, ('Your profile was successfully updated!'))
+            return redirect('home')
+        else:
+            print profile_form.errors
+            messages.error(request, ('Please correct the error below.'))
```
Comment (ananthu-nair707) [inline profiles/views.py:30]: Remove debug statements

File: profiles/forms.py (hunk 1 above)
Comment (simranjits-cuelogic) [reply profiles/forms.py]: Code removed. As we don't have requirement for validating image as of now.

File: profiles/views.py (hunk 2 above)
Comment (simranjits-cuelogic) [reply profiles/views.py:21]: Doc string provided explaining the purpose and working of code.

File: profiles/views.py (hunk 3 above)
Comment (simranjits-cuelogic) [reply profiles/views.py:30]: Debug statement removed.

**Pull request merged**